src/index.html

```diff
@@ -17,25 +17,19 @@ <h1 class="title">Leaderboard </h1>
     <div class="scores-section">
       <div class="recent">
         <h4>Recent scores</h4>
-        <button class="btn">Refresh</button>
+        <button class="btn" id="refresh-btn">Refresh</button>
       </div>
-      <div class="recent-scores">
-        <p>Name: 100</p>
-        <p>Name: 20</p>
-        <p>Name: 50</p>
-        <p>Name: 78</p>
-        <p>Name: 125</p>
-        <p>Name: 77</p>
-        <p>Name: 42</p>
+      <div class="recent-scores" id="all-scores">
+        
       </div>
     </div>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231142810-0e0b91c4-dd3a-4735-85e6-1961fc34de2b.png)

**[OPTIONAL]**

- [ ] It would be very nice if you put `npm start` command in the Usage section to make your readme file descriptive, useful and professional 👍